### ISSUE TITLE
NTR: Malaria parasite in thick peripheral blood smear (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8072]
New term request Malaria parasite in thick peripheral blood smear (samant_110129125209): Plasmodium

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000553
- label: Bloodstream Malaria parasite
- parents: [Blood parasite (HP:0034523)](https://hpo.jax.org/app/browse/term/HP:0034523)
- pmid: PMID:35255896
- def: Presence of malaria parasites in the peripheral blood.
- comment: The analysis of thick blood smears (TBS) is the reference method chosen as a first option for the malaria diagnosis worldwide. In this method, after a blood sample is taken and dried, the TBS is stained. After the staining process, the smear is visually analysed by microscopy, where parasites and leukocytes are identified and counted.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8072
- synonyms:
